spec/models/preserved_object_spec.rb

```diff
@@ -199,4 +199,39 @@
       expect(described_class.daily_check_count).to eq 11
     end
   end
+
+  describe '#total_size_of_moab_version' do
+    let(:druid) { 'bz514sm9647' }
+    let(:preserved_object) { create(:preserved_object, druid: druid, current_version: current_version) }
+    let(:current_version) { 3 }
+
+    context 'when complete moab is nil' do
+      it 'returns 0' do
+        expect(preserved_object.total_size_of_moab_version(current_version)).to eq(0)
+      end
+    end
+
+    context 'when complete moab exists' do
+      let!(:cm1) { create(:complete_moab, preserved_object: preserved_object, version: current_version, moab_storage_root: msr) } # rubocop:disable RSpec/LetSetup
```

Comment:
Maybe I failed to drink enough 🍵 before working on this, but I couldn't tell why Rubocop was dinging me for `let!` on this line when this file uses plenty of `let!` already. I found 0 other references to `RSpec/LetSetup` in the codebase, when I expected to see this cop disabled or excluded elsewhere. Alas. 🤷🏻‍♂️

Reply:
I *think* it might have been complaining because `cm1` is never explicitly called.  If this is done in a before block, without using let, I think it would work.  but no biggie.